x/bounty/keeper/grpc_query.go

```diff
@@ -68,6 +67,10 @@ func (k Keeper) Findings(c context.Context, req *types.QueryFindingsRequest) (*t
 	var queryFindings types.Findings
 	ctx := sdk.UnwrapSDKContext(c)
 
+	if len(req.ProgramId) == 0 && len(req.SubmitterAddress) == 0 {
```

Comment:
So we don't support query all findings on chain ?

Reply:
I don't think it's necessary to support all finding output.

Reply:
Yes, one can query all programs and iterate programID to get all findings